docs/source/sensors.rst

```diff
@@ -192,6 +192,90 @@ API Docs
 .. autoclass:: st2reactor.container.sensor_wrapper.SensorService
     :members:
 
+Partitioning Sensors
```

Comment:
Reasonable. I thought about it as well but could not work out where to add this relatively advanced topic. I am happy to change.